Build.fsx

```diff
@@ -244,14 +243,13 @@ Target "PublishNuGetAll" DoNothing
 "PatchAssemblyVersions" ==> "Build"
 "BuildOnly"             ==> "Build"
 
+"BuildOnly" 
+    ==> "TestOnly"
+    ==> "BuildAndTestOnly"
+
 "Build"    ==> "Test"
 "TestOnly" ==> "Test"
 
-"BuildOnly" ==> "TestOnly"
-
-"BuildOnly" ==> "BuildAndTestOnly"
-"TestOnly"  ==> "BuildAndTestOnly"
-
```

Comment:
It depends on the usage scenario. From the `CompleteBuild` or `AppVeyor` perspective, they are not required. However, personally I find the `BuildOnly` and `TestOnly` targets quite useful. They allows to _quickly_ check whether project compiles now and tests pass (these targets don't have dependencies on cleanup and validations). I used them for a couple of times.

Also these targets were present since the script creation, so might be used by people. I'd vote to keep them given that they make sense and are not useless 😉